pcdsdevices/epics_motor.py

```diff
@@ -54,12 +55,36 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
 
     # Enable/Disable puts
     disabled = Cpt(EpicsSignal, ".DISP", kind='omitted')
+    set_metadata(disabled, dict(variety='command-enum'))
     # Description is valuable
     description = Cpt(EpicsSignal, '.DESC', kind='normal')
 
     tab_whitelist = ["set_current_position", "home", "velocity",
                      "enable", "disable"]
 
+    set_metadata(EpicsMotor.home_forward, dict(variety='command-proc', value=1))
+    EpicsMotor.home_forward.kind = Kind.normal
+    set_metadata(EpicsMotor.home_reverse, dict(variety='command-proc', value=1))
+    EpicsMotor.home_reverse.kind = Kind.normal
+    set_metadata(EpicsMotor.low_limit_switch, dict(variety='bitmask', bits=1))
+    EpicsMotor.low_limit_switch.kind = Kind.normal
+    set_metadata(EpicsMotor.high_limit_switch, dict(variety='bitmask', bits=1))
+    EpicsMotor.high_limit_switch.kind = Kind.normal
+    set_metadata(EpicsMotor.motor_done_move, dict(variety='bitmask', bits=1))
+    EpicsMotor.motor_done_move.kind = Kind.config
+    set_metadata(EpicsMotor.motor_is_moving, dict(variety='bitmask', bits=1))
+    # Had no joy here or anywhere else with enums. I still get the <val> <egu> 
+    # "0 mm" readback. Maybe I'm doing it wrong?
+#    set_metadata(EpicsMotor.motor_is_moving, dict(variety='enum',
```

Comment:
@klauer I would have thought that this would have worked. However, I didn't see anyone else using enum_strings in pcdsdevices; maybe this isn't totally working yet?

Reply:
The code at https://github.com/pcdshub/typhos/blob/674686b5b93c42352c5c0902f5684a9fd75bac04/typhos/widgets.py suggests that the methods to use the `enum_strings` are not yet written for `enum` or `text-enum`. Only the command varieties have handling for this.

In general the variety handling is not complete- we can justify working on this now that someone wants to take full advantage of it.

Reply:
Just confirming @ZLLentz's comment - it's just not there quite yet.